media-service/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'java'
+}
+
+group = 'org.arpha'
+version = '1.0.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project (':common')
+    implementation project (":abstract-media-service")
+    implementation project (":user-service")
+    testImplementation platform('org.junit:junit-bom:5.10.0')
+    testImplementation 'org.junit.jupiter:junit-jupiter'
+}
+
+test {
+    useJUnitPlatform()
+}
```

Comment:
Please make this build.gradle empty. This file should contain only project dependencies. Everything else will be inherited from base build.gradle